### ISSUE TITLE
Add cs icon support for leaderboards 

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#1332]
![image](https://github.com/user-attachments/assets/c4a95c18-0b1a-4fcf-9352-54b3f2529db4)
would be hugely cool.

**Comment by EmeraldLoc** [comment]
Dont know if theres an api for this, can see

**Comment by TheMan6900** [comment]
Cs does support icons so 👍

**Comment by Bear64DX** [comment]
I think this is actually a thing in the newest Dynamic Life Icons update, as well as Toad's new icon (tag uses an outdated version 
of Dynamic Icons iirc)

**Comment by EmeraldLoc** [comment]
I updated dynamic life icons recently, and the new toad icon has been added. I can see if this is true or not.

**Comment by EmeraldLoc** [comment]
Yes this does indeed exist, I will implement this now if its not already implemented
